connector_easypost/unit/import_synchronizer.py

```diff
@@ -137,8 +140,15 @@ def _get_binding(self):
                                    unwrap=False,
                                    browse=True)
 
-    def _create_data(self, map_record, **kwargs):
-        return map_record.values(for_create=True, **kwargs)
+    def _generate_easypost_id(self):
+        """ Some objects in EasyPost have no ID. Return one based on
+        the unique data of the object
+        :return str: The MD5 Hex Digest of the record
+        """
+        obj_hash = md5()
+        for attr in self._hashable_attrs:
+            obj_hash.update(getattr(self.easypost_record, attr, '') or '')
+        return '%s_%s' % (self._id_prefix, obj_hash.hexdigest())
```

Comment:
Nah, it needs to be unique to the data of the object. That way, if a duplicate object without an ID is created, it can be matched up as a pre-existing record.